content/20.pharmaceuticals.md

```diff
@@ -870,7 +869,7 @@ Over the longer term, this approach and others may lead to the development of no
 | Bamlanivimab and etesevimab | Biologic, monoclonal antibodies | EUA | | | |
 | SNG001 | Biologic, interferon | None | RCT | Mixed results: support from initial RCT but no effect found in WHO's Solidarity trial |
 
-Table: Summary table of candidate therapeutics examined in this manuscript. The FDA status is provided where available. The evidence available is based on the progression of the therapeutic through the pharmaceutical development pipeline, with randomized control trials (RCT) as the most informative source of evidence. The effectiveness is summarized based on the current available evidence; large trials such as RECOVERY and Solidarity are weighted heavily in this summary. {#tbl:summary-table}
+Table @tbl:summary-table: Summary table of candidate therapeutics examined in this manuscript. The FDA status is provided where available. The evidence available is based on the progression of the therapeutic through the pharmaceutical development pipeline, with randomized control trials (RCT) as the most informative source of evidence. The effectiveness is summarized based on the current available evidence; large trials such as RECOVERY and Solidarity are weighted heavily in this summary. This table was last updated on February 17, 2021. {#tbl:summary-table}
```

Comment:
```suggestion
Table: Summary table of candidate therapeutics examined in this manuscript. The FDA status is provided where available. The evidence available is based on the progression of the therapeutic through the pharmaceutical development pipeline, with randomized control trials (RCT) as the most informative source of evidence. The effectiveness is summarized based on the current available evidence; large trials such as RECOVERY and Solidarity are weighted heavily in this summary. This table was last updated on February 17, 2021. {#tbl:summary-table}
```

The way you had it initially is working  The table number gets added by the Pandoc filter:
![image](https://user-images.githubusercontent.com/3532898/108513957-dd5e5e00-7288-11eb-98c2-0152687ffcc2.png)

Reply:
Oh wow! So cool!